### DISCUSSION TITLE
Fatal error after bin/update

### DISCUSSION THREAD

**Discussion by danathala** [markshust/docker-magento#436] [Q&A]
Fatal error: Uncaught Error: Call to undefined function xdebug_disable() in /var/www/html/vendor/magento/magento2-functional-testing-framework/src/Magento/FunctionalTestingFramework/_bootstrap.php:81

tried to remove this Line - xdebug_disable but nothing works. The issue happened after I ran bin/update

after bin/update got the above error. Running php version 7.4 

I tired to add this fix but nothing works

vendor/magento/magento2-functional-testing-framework/src/Magento/FunctionalTestingFramework/_bootstrap.php

From :-

if (!(bool)$debugMode && extension_loaded('xdebug')) {
    xdebug_disable();
}
To :-

if (!(bool)$debugMode && extension_loaded('xdebug')) {
    if (function_exists('xdebug_disable')) {
        xdebug_disable();
    }
}

**Comment by piotrkwiecinski** [comment]
@danathala that has been fixed in testing framework some time ago https://github.com/magento/magento2-functional-testing-framework/blob/develop/CHANGELOG.md#321

Try updating the module: `magento/magento2-functional-testing-framework` to 3.2.1+

**Reply by danathala** [reply]
HI @piotrkwiecinski I blown up the old installation and was trying to install from start magento 2.4 but unfortunately
I ended up with an error my process is getting killed on waiting for connection to elastic search (using MacBook Air not with M1 chip)

Your system password has been requested to add an entry to /etc/hosts...
Password:
127.0.0.1 ::1 magento.test
Stopping m242_app_1           ... done
Stopping m242_phpfpm_1        ... done
Stopping m242_db_1            ... done
Stopping m242_elasticsearch_1 ... done
Stopping m242_rabbitmq_1      ... done
Stopping m242_redis_1         ... done
Starting m242_redis_1         ... done
Starting m242_elasticsearch_1 ... done
Starting m242_rabbitmq_1      ... done
Starting m242_db_1            ... done
Starting m242_phpfpm_1        ... done
Starting m242_app_1           ... done
Waiting for connection to Elasticsearch...
......ERROR: 124
dhanasekarp@Dhanasekars-Air m242 %

**Reply by danathala** [reply]
@piotrkwiecinski can you please help me on this

**Comment by danathala** [comment]
echo "Waiting for connection to Elasticsearch..."
bin/clinotty timeout 30 bash -c " error fixed after increasing the value from 30 -100 on bin/setup file
